store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java

```diff
@@ -424,6 +424,26 @@ public class MessageStoreConfig {
 
     private boolean putConsumeQueueDataByFileChannel = true;
 
+    private boolean transferOffsetJsonToRocksdb = false;
+
+    private boolean rocksdbCQWriteEnable = false;
```

Comment:
The naming need to be clearer, for example, by emphasizing double writing and adding comments, otherwise users may confuse it with the parameters for opening rocksdb.